src/javascript/binary/websocket_pages/user/account/profit_table/profit_table.init.js

```diff
@@ -106,18 +102,12 @@ const ProfitTable = (() => {
         if (opts) $.extend(true, req, opts);
 
         BinarySocket.send(req).then((response) => {
-            ProfitTable.profitTableHandler(response);
+            ProfitTableInit.profitTableHandler(response);
```

Comment:
is this referring to a local function? if so it doesn't need to call the module name